application/src/main/java/net/corda/v5/application/interop/facade/FacadeReader.java

```diff
@@ -0,0 +1,34 @@
+package net.corda.v5.application.interop.facade;
+
+import org.jetbrains.annotations.NotNull;
+import java.io.*;
+
+public interface FacadeReader {
+    /**
+     * Read a [KotlinFacade] from the given [Reader].
```

Comment:
Is KotlinFacade the best name or would it be confusing ?

Reply:
I'm writing a java cordapp, do i need a javafacade?

Reply:
Typo, should be just "Facade"